server/datastore/mysql/migrations/tables/20220608113128_AddTransparencyURLToAppConfig_test.go

```diff
@@ -0,0 +1,33 @@
+package tables
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20220608113128(t *testing.T) {
```

Comment:
Input on whether this testing approach makes sense would be very much appreciated